.github/workflows/ci.yml

```diff
@@ -61,3 +62,4 @@ jobs:
         with:
           working-directory: ./websites/visualization
           comment-strategy: 'always'
```

Comment:
[`skip-insignificant`](https://github.com/transferwise/actions-next-bundle-analyzer?tab=readme-ov-file#usage) is skip commenting on PRs if there are no signficant changes in page sizes.

Do you think we should always show the bundle size of the visualization website, even if there's no change in size?